home-data-migration/src/main/resources/db/changelog/0.0.1/Issue33.xml

```diff
@@ -5,35 +5,6 @@
         xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
                       http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
 
-    <changeSet id="Issue33.001" author="Arthurpanda">
-        <createTable tableName="users">
-            <column name="ID" type="BIGINT">
-                <constraints primaryKey="true" nullable="false"/>
-            </column>
-            <column name="first_name" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="last_name" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="email" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="password" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="contact" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="flat" type="VARCHAR(255)">
-                <constraints nullable="false"/>
-            </column>
-            <column name="active" type="boolean">
-                <constraints nullable="false"/>
-            </column>
-        </createTable>
-    </changeSet>
-
```

Comment:
Why do you delete this table?

Reply:
Because we are not gonna use it so it's dead code as Aleksandr called it.